### ISSUE TITLE
valid-jsdoc incorrectly flags multi-level params as errors

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#925]
Given JSDoc such as this:

``` js
/**
 * Traverses multiple directories and calls a callback on each file.
 * @param {Object} options The option for the traversal.
 * @param {string[]} options.files An array of file and directory paths to traverse.
 * @param {string[]} options.exclude An array of file and directory paths to ignore.
 * @param {Function} callback A function to call for each file.
 * @returns {void}
 */
```

The output is:

```
  83:0  error  Expected JSDoc for 'callback' but found 'options.files'  valid-jsdoc
```

We should skip over any named parameters with a dot, as these are properties of an object that is a parameter. There should be no warning in this case.
